clang/lib/Sema/SemaLambda.cpp

```diff
@@ -1323,7 +1323,6 @@ void Sema::ActOnLambdaExpressionAfterIntroducer(LambdaIntroducer &Intro,
 
     if (C->Init.isUsable()) {
       addInitCapture(LSI, cast<VarDecl>(Var), C->Kind == LCK_ByRef);
-      PushOnScopeChains(Var, CurScope, false);
```

Comment:
Yeah, 2 is a duplicate of 1, but 3 is necessary because that constitutes a lambda body scope.